### PR TITLE
Add support for `bloop server`

### PR DESCRIPTION
**Pull request by jvican**
Adds improvements in nailgun that fix https://github.com/scalacenter/bloop/issues/320.

### CODE REVIEW

**Comment by jvican** [comment]
CI error in https://github.com/scalacenter/bloop/pull/400/commits/53de32c19e3d61bb301ee4e1ca43d3244e02fd46 is spurious.

File: website/content/docs/how-to-use-bloop.md [hunk 1]
```diff
@@ -10,22 +10,57 @@ bref = "Learn how to use bloop and get familiar with its client-server architect
...
+
+<span class="label warning">Note</span> There can only be one server running
+per user session. If you want to run more than one server (strongly not
+recommended unless you know what you're doing), then specify the nailgun port
+in both the server and the client.
+
+The server is installed by default in the same directory than the `bloop`
+client, so `bloop server` will try to find the location of the server in the
+same directory where `bloop` is installed. If this is not the case, run `bloop
+---server-location=/foo/my-blp-server server` instead.
```
Comment (Duhemm) [inline website/content/docs/how-to-use-bloop.md]: Are there 3 `-` on purpose?

File: website/content/docs/how-to-use-bloop.md [hunk 2]
```diff
@@ -10,22 +10,57 @@ bref = "Learn how to use bloop and get familiar with its client-server architect
...
+
+Have you forgotten to start bloop's server? Run it with `bloop server`.
+Check our usage instructions in https://scalacenter.github.io/bloop/
+
+Type `--nailgun-help` for help on the Nailgun CLI tool.
+```
+
+The bloop server can serve responses of many concurrent clients.
+
+#### Example
```
Comment (Duhemm) [inline website/content/docs/how-to-use-bloop.md]: I think that we should get rid of this paragraph. It doesn't bring much, and the given example is not a runnable command (no project is specified).

**Comment by jvican** [comment]
> Maybe that we should also use bloop server to start the service in the homebrew formula?
  
`bloop server` is meant to be used by the users, so I don't see much value in doing that.
  
Feedback addressed.

**Pull request merged**